execute.go

```diff
@@ -217,10 +217,17 @@ loop:
 				break loop
 			}
 			backtrack = false
+
+			var xc [2]interface{}
+			var xv interface{}
```

Comment:
Idea was to refactor so that the pus/fork below can be shaed. `xc` is current and `xn` is next for `opeach`.